source/_integrations/dlna_dmr.markdown

```diff
@@ -49,3 +49,11 @@ DLNA devices can support a range of features. Depending on the device itself, th
 * `media_player.shuffle_set`
 * `media_player.repeat_set`
 * `media_player.select_sound_mode`
+
+## Playing media
+
+Most DLNA DMR devices can play media from local HTTP servers. For best results, use HTTP instead of HTTPS, and refer to the server using an IP address instead of a host name, e.g. `http://192.168.1.1:8123/song.mp3`.
```

Comment:
Perhaps the port `8123` shouldn't be used. This seems to imply that hass can serve media files. Either don't specify a port or use a more generic port such as 8080?

Reply:
Heh, it can to an extent, but not this easily. Fixed.